config_defaults_inc.php

```diff
@@ -2579,6 +2579,71 @@
 	'view_state',
 );
 
+/**
+ * An array of access levels to identify who should be presented a simplified report form.
+ *
+ * The following Statuses are standard available:
+ *   - VIEWER
+ *   - REPORTER
+ *   - UPDATER
+ *   - DEVELOPER
+ *   - MANAGER
+ *   - ADMINISTRATOR
+ *
+ * @global array $g_simple_report_level
+ */
+$g_simple_report_level = array( REPORTER, UPDATER, );
```

Comment:
As mentioned before, this can now be set to `$g_simple_report_level = UPDATER;`

Reply:
While more thinking about that, I would change the name of the variable to `$g_simple_report_levels` and set it to `NOBODY` instead of `UPDATER`.

@CasN as response to your question submitted vie email: I asked for the change from array to integer, as it's the way it's done for quite a lot of similar configuration options (`$g_*_threshold`)
Of course, it's not consistent to have "_threshold_" as part of a variable name and to use it not just for integer access level threshold, but also for arrays of access levels.
The reason for that is, that the ability to use arrays in addition to integer values was introduced after naming.
The right clean way would be to rename all these options, deprecate/obsolete them, migrate them, ...
This is certainly nothing that should be done as part of the current PR.
It would have to be implemented  as part of a bigger code cleanup when introducing a new MantisBT major 3.x version, as it would break some 3rd party plugins.